pyanaconda/ntp.py

```diff
@@ -143,7 +152,16 @@ def get_servers_from_config(conf_file_path=NTP_CONFIG_FILE):
                 server = TimeSourceData()
                 server.type = match.group(1).upper()
                 server.hostname = match.group(2)
-                server.options = ["iburst"]
+                server.options = []
+
+                words = match.group(3).lower().split()
+                for i in range(len(words)):
+                    if words[i] in SRV_NOARG_OPTIONS:
+                        server.options.append(words[i])
+                    elif words[i] in SRV_ARG_OPTIONS and i + 1 < len(words):
+                        server.options.append(' '.join(words[i:i+2]))
+                    # ignore unknown options
```

Comment:
Maybe we can log those unknown options ? That would also make the comment look better and not a bit dangling like this.

Reply:
Makes sense. Thanks.